setup.md

```diff
@@ -16,3 +16,50 @@ $ cd
 {: .bash}
 
 In the lesson, you will find out how to access the data in this folder.  
+
+> ## Where to Type Commands: How to Open and Use a Unix Shell Command-Line Interface program
```

Comment:
I think the terminology should be tightened here. Maybe 
> How to open a new shell?

Then as the first line after the heading, you could explain that this might be referred to as the `Terminal` or `Command line`